include/inexor/vulkan-renderer/wrapper/glfw_context.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <GLFW/glfw3.h>
+#include <spdlog/spdlog.h>
```

Comment:
Both glfw or spdlog includes can be moved to the source file.

Reply:
Right.

Reply:
Done.